bin/hdfcoinc/pycbc_coinc_findtrigs

```diff
@@ -239,94 +252,153 @@ for tnum in template_ids:
     logging.info('Calculating Single Detector Statistic')
     s0g, s1g = rank_method.single(trigs0), rank_method.single(trigs1)
 
+    try:
+        s0gstat = s0g['snglstat'].copy()
```

Comment:
When would this work/not work?

Reply:
This wouldn't work if you're using newsnr, as in that case s0g is just a singe array.

Reply:
Okay, would you be able to add a comment to say that "s0g is sometimes an array of arrays, sometimes just a single array"

Reply:
A comment has been added to explain this